formula/nwchem.rb

```diff
@@ -0,0 +1,85 @@
+class Nwchem < Formula
+  desc "NWChem: Open Source High-Performance Computational Chemistry"
+  homepage "http://www.nwchem-sw.org"
+  url "https://github.com/nwchemgit/nwchem/releases/download/6.8.1-release/nwchem-6.8.1-release.revision-v6.8-133-ge032219-src.2018-06-14.tar.bz2"
+  version "6.8.1"
+  sha256 "23ce8241a5977a93d8224f66433851c81a08ad58a4c551858ae031485b095ab7"
+
+  depends_on "openblas"
+  depends_on "gcc" # for gfortran
+  depends_on "open-mpi"
+  depends_on "scalapack"
+
+  def install
+    cd "QA" do
+      (prefix/"QA").mkdir
+      (prefix/"QA/tests").mkdir
+      cp "runtests.mpi.unix", prefix/"QA"
+      cp "nwparse.pl", prefix/"QA"
+      cp_r "tests/dft_he2+", prefix/"QA/tests"
+      cp_r "tests/prop_mep_gcube", prefix/"QA/tests"
+      cp_r "tests/tce_n2", prefix/"QA/tests"
+      cp_r "tests/tddft_h2o", prefix/"QA/tests"
+      cp_r "tests/pspw", prefix/"QA/tests"
+    end
+
+    cd "src" do
+      (prefix/"etc").mkdir
+      (prefix/"etc/nwchemrc").write <<~EOS
+        nwchem_basis_library #{share}/libraries/
+        nwchem_nwpw_library #{share}/libraryps/
+        ffield amber
+        amber_1 #{share}/amber_s/
+        amber_2 #{share}/amber_q/
+        amber_3 #{share}/amber_x/
+        amber_4 #{share}/amber_u/
+        spce    #{share}/solvents/spce.rst
+        charmm_s #{share}/charmm_s/
+        charmm_x #{share}/charmm_x/
+      EOS
+
+      inreplace "util/util_nwchemrc.F", "/etc/nwchemrc", "#{etc}/nwchemrc"
+      inreplace "config/makefile.h" do |s|
+        s.gsub! /-mtune=native/, "-mtune=generic"
+        s.gsub! /-mfpmath=sse/, " "
+        s.gsub! /-msse3/, " "
+      end
+
+      ENV["NWCHEM_TOP"] = "#{Dir.pwd}/.."
+      ENV["PYTHONVERSION"] = "2.7"
```

Comment:
Pretty much all of this are things upstream should be detecting themselves rather than having to be manually fed every possible chunk of information about the build environment & dependency fulfilment.